src/networks.js

```diff
@@ -14,6 +15,7 @@ module.exports = {
     dustThreshold: 546 // https://github.com/bitcoin/bitcoin/blob/v0.9.2/src/core.h#L151-L162
   },
   testnet: {
```

Comment:
@jprichardson I suppose,  just thinking about avoiding changes in some unforseeable future change